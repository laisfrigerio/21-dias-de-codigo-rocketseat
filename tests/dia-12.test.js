const { JSDOM } = require('jsdom')
const fs = require('fs')
const path = require('path')

const html = fs.readFileSync(path.resolve(__dirname, '../dia-12/index.html'), 'utf8')

let dom
let container

describe('dia 12', () => {
  beforeEach(() => {
    dom = new JSDOM(html, { runScripts: 'dangerously' })
    container = dom.window.document.body
  })

  test('should match the gallery images challenge', () => {
    expect(container).toMatchSnapshot();
  });
})