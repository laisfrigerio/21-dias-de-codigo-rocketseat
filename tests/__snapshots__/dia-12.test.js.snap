// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dia 12 should match the gallery images challenge 1`] = `
<body
  id="particles-js"
>
  
  
  <header />
  

  
  <section
    class="gallery"
  >
    
    
    <figure
      class="card-img"
    >
      
      
      <img
        alt=""
        src="../assets/images/gallery/dia-02-hello-world.png"
      />
      
      
      <figcaption
        class="legend"
      >
        Dia 02: Hello World
      </figcaption>
      
    
    </figure>
    
    
    <figure
      class="card-img"
    >
      
      
      <img
        alt=""
        src="../assets/images/gallery/dia-03-botao.png"
      />
      
      
      <figcaption
        class="legend"
      >
        Dia 03: Botão animado
      </figcaption>
      
    
    </figure>
    
    
    <figure
      class="card-img"
    >
      
      
      <img
        alt=""
        src="../assets/images/gallery/dia-04-calculadora.png"
      />
      
      
      <figcaption
        class="legend"
      >
        Dia 04: Calculadora
      </figcaption>
      
    
    </figure>
    
    
    <figure
      class="card-img"
    >
      
      
      <img
        alt=""
        src="../assets/images/gallery/dia-05-erro-404.png"
      />
      
      
      <figcaption
        class="legend"
      >
        Dia 05: Página 404
      </figcaption>
      
    
    </figure>
    
    
    <figure
      class="card-img"
    >
      
      
      <img
        alt=""
        src="../assets/images/gallery/dia-06-profile-card.png"
      />
      
      
      <figcaption
        class="legend"
      >
        Dia 06: Glassmorphism
      </figcaption>
      
    
    </figure>
    
    
    <figure
      class="card-img"
    >
      
      
      <img
        alt=""
        src="../assets/images/gallery/dia-07-contador.png"
      />
      
      
      <figcaption
        class="legend"
      >
        Dia 07: Contador
      </figcaption>
      
    
    </figure>
    
    
    <figure
      class="card-img"
    >
      
      
      <img
        alt=""
        src="../assets/images/gallery/dia-08-tema.png"
      />
      
      
      <figcaption
        class="legend"
      >
        Dia 08: Toggle tema
      </figcaption>
      
    
    </figure>
    
    
    <figure
      class="card-img"
    >
      
      
      <img
        alt=""
        src="../assets/images/gallery/dia-09-loading.png"
      />
      
      
      <figcaption
        class="legend"
      >
        Dia 09: Loading
      </figcaption>
      
    
    </figure>
    
    
    <figure
      class="card-img"
    >
      
      
      <img
        alt=""
        src="../assets/images/gallery/dia-10-particles-js.png"
      />
      
      
      <figcaption
        class="legend"
      >
        Dia 10: particle.js
      </figcaption>
      
    
    </figure>
    
    
    <figure
      class="card-img"
    >
      
      
      <img
        alt=""
        src="../assets/images/gallery/dia-11-login.png"
      />
      
      
      <figcaption
        class="legend"
      >
        Dia 11: Formulário Login
      </figcaption>
      
    
    </figure>
    
  
  </section>
  
  
  <section
    class="modal"
  >
    
    
    <div
      class="close-btn"
    >
      ×
    </div>
    
    
    <div
      class="modal-content"
    >
      
      
      <img
        alt=""
        src="../assets/images/gallery/dia-05-erro-404.png"
      />
      
    
    </div>
    
  
  </section>
  
  
  <footer />
  
  
  <script
    src="../assets/js/particles.js"
  />
  
  
  <script
    src="../assets/js/include-html.js"
  />
  
  
  <script
    src="scripts.js"
  />
  
  
  <script
    src="https://cdn.jsdelivr.net/npm/particles.js@2.0.0/particles.min.js"
  />
  



</body>
`;
